Earthfile

```diff
@@ -601,44 +526,16 @@ netboot:
     SAVE ARTIFACT /build/$ISO_NAME-initrd initrd AS LOCAL build/$ISO_NAME-initrd
     SAVE ARTIFACT /build/$ISO_NAME.ipxe ipxe AS LOCAL build/$ISO_NAME.ipxe
 
-artifact-name:
-    ARG TARGETARCH
-    ARG --required FAMILY
-    ARG --required FLAVOR
-    ARG --required FLAVOR_RELEASE
-    ARG --required VARIANT
-    ARG --required MODEL
-    ARG --required BASE_IMAGE
-    ARG --required NAMING_FUNC
-    ARG --required NAMING_EXT
-    ARG --required KAIROS_VERSION
-    FROM ubuntu
-
-    COPY ./images/naming.sh /usr/bin/local/naming.sh
-    RUN echo $(/usr/bin/local/naming.sh ${NAMING_FUNC})${NAMING_EXT} > /ARTIFACT_NAME
-    SAVE ARTIFACT /ARTIFACT_NAME ARTIFACT_NAME
-
 arm-image:
   ARG OSBUILDER_IMAGE
   ARG COMPRESS_IMG=true
   ARG IMG_COMPRESSION=xz
-  FROM $OSBUILDER_IMAGE
 
-  COPY +version/VERSION ./
-  ARG KAIROS_VERSION=$(cat VERSION)
-  RUN echo "version ${KAIROS_VERSION}"
+  FROM +base-image
```

Comment:
this is the cause for the alpine not to build, because it is re-running the base image but on x86, so it doesn't find the rpi packages. I will try to produce an artifact and just consume that instead of doing this change